docs/platform/manager/accounts-and-passwords-new-manager/index.md

```diff
@@ -0,0 +1,162 @@
+7.  To give the new user restricted access, toggle the **Restricted Access** button.
+8.  Click **Submit**.
+
+If you granted the user full access, the account will be created and no further action is required. If you opted to restrict the user, follow the instructions in the next section.
+
+### Setting Permissions
+
+Setting permissions restricts a user's access to certain areas of the Linode Manager. For example, you could limit a user to a single Linode and prevent them from removing the Linode or adding extra services. Don't worry--these settings aren't permanent. You can add or remove access for a user at any time in the future.
+
+Here's how to set a user's access permissions:
+
+1.  Click the **Account** link on the left-hand side of the page.
+2.  Click the **Users** link.
+3.  Click on the user in the list and select the **User Permissions** tab. The webpage shown below appears.
+
+    [![Configure User permissions in the Linode Manager.](accounts-user-permissions-small.png "Configure User permissions in the Linode Manager")](accounts-user-permissions.png)
+
+4.  Toggle the boxes in the **Global Permissions** section to allow the user to add [Linodes](/docs/getting-started/), [NodeBalancers](/docs/platform/nodebalancer/getting-started-with-nodebalancers/), [Domains](/docs/networking/dns/dns-manager-overview/#domain-zones), [Longview](/docs/platform/longview/longview/) clients, [Block Storage Volumes](/docs/platform/block-storage/how-to-use-block-storage-with-your-linode/) to the account, create [StackScripts](/docs/platform/stackscripts/) and frozen [Images](/docs/platform/disk-images/linode-images/), access all billing information, and cancel the entire account.
+
+    {{< note >}}
+Granting access to settings denoted with a dollar sign ($) will allow the user to perform actions that incur billing costs, such as adding or resizing a Linode.
+{{< /note >}}
+
+1.  Select the appropriate permissions (None, Read Only, or Read-Write) in the **Specific Grants** section to allow the user access your Linodes, StackSripts, Images, NodeBalancers and Domains.
+2.  When you have finished configuring the user's permissions, click **Save**. The user's permissions will be saved and effective immediately.
+
+### Recovering a Lost Username
+
+Did you forget your Linode Manager username? Recover it with the *Forgot Username* webpage. Here's how:
+
+1.  Visit the [Forgot Username](https://manager.linode.com/session/forgot/username) webpage.
+1.  Enter your email address in the **Email** field.
+1.  Click **Submit**.
+
+In a couple minutes, you'll receive an email message with any Linode Manager users that correspond to that email. If you do not receive information about any users, then you may have registered your account with a different email.
+
+### Removing a User
+
+You can permanently remove a user account from the Linode Manager. Here's how:
```

Comment:
this is also possible from the user's profile view. Not sure if it's worth mentioning here.